navigation/MainNavigation.js

```diff
@@ -39,12 +39,14 @@ import ResetPasswordScreen from '../screens/ResetPasswordScreen';
 import WelcomeScreen from '../screens/WelcomeScreen';
 import ConnectionsScreen from '../screens/Connections/ConnectionsScreen';
 import SelectedConnectionsScreen from '../screens/Connections/SelectedConnectionsScreen';
+import SkillRequestScreen from '../screens/SkillRequestScreen';
```

Comment:
Can we call this tab the "Skilled Impact" tab? IMO this makes it sound like it's only for applications when there's also a lot of profile config, groups, etc.

So can we basically just replace `SkillRequest` with `SkilledImpact` everywhere in this file?

Reply:
You mean call this screen the "SkilledImpactScreen"? Just for clarification.

Reply:
Yeah I think that would be best. As well as renaming `SkillRequestsStack` to `SkilledImpactStack`, etc.

Reply:
Got it. Will do.